src/com/lftechnology/java/training/sanish/numberprinter/NumberPrinter.java

```diff
@@ -0,0 +1,32 @@
+package com.lftechnology.java.training.sanish.numberprinter;
+
+import java.util.logging.Level;
+import java.util.logging.Logger;
+
+/**
+ * Print number for 1-10
+ *
+ * @author Sanish Maharjan <sanishmaharjan@lftechnology.com>
+ */
+public class NumberPrinter {
+    private static final Logger LOGGER = Logger.getLogger(NumberPrinter.class.getName());
+
+    public static void main(String[] agrs) {
+        System.out.print("Hello world.");
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Replace this usage of System.out or System.err by a logger. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS106)